packaging/nnstreamer.spec

```diff
@@ -179,6 +179,22 @@ Developmental kit for Tizen Native new single-shot NNStreamer API.
 %define api -Denable-capi=false
 %endif
 
+# Element restriction in Tizen
+%if %{with tizen}
+%define restricted_element	'capsfilter input-selector output-selector queue tee valve appsink appsrc audioconvert audiorate audioresample videoconvert videocrop videorate videoscale fakesrc fakesink filesrc filesink audiotestsrc videotestsrc jpegparse jpegenc jpegdec pngenc pngdec tcpclientsink tcpclientsrc tcpserversink tcpserversrc udpsink udpsrc xvimagesink ximagesink evasimagesink evaspixmapsink glimagesink theoraenc lame vorbisenc wavenc volume oggmux avimux matroskamux avsysvideosrc camerasrc fimcconvert'
```

Comment:
`videomixer` to support overlay of videos together?
Useful for displaying the results of object recognition.

Reply:
`videoflip` to adjust orientation of incoming input video for tizen/andriod applications.